File: testresources/csharp/benchmark/Domain/Direct/Violating/AccessObjectReferenceWithinIfStatement.cs
namespace Domain.Direct.Violating
{
	using Domain.Direct;	
	using Technology.Direct.Dao;

	public class AccessObjectReferenceWithinIfStatement
	{
		private ProfileDAO profileDao;

		public AccessObjectReferenceWithinIfStatement()
		{
			if (profileDao != null)
			{
                string s = "wrong";
			}
		}
	}
}

File: testresources/java/accuracy/src/domain/direct/violating/AccessObjectReferenceWithinIfStatement.java
package domain.direct.violating;

import technology.direct.dao.ProfileDAO;

public class AccessObjectReferenceWithinIfStatement {
	
	private ProfileDAO profileDao;

	public AccessObjectReferenceWithinIfStatement(){
		if (profileDao != null) {
			String s = "wrong";
			System.out.println(s);
		}
	}
}